module.json

```diff
@@ -99,6 +100,10 @@
 			{
 				"id": "worldbuilding",
 				"type": "system"
+			},
+			{
+				"id": "ac2d20",
+				"type": "system"
```

Comment:
Can you move this so the list in is alphabetical order please?

Reply:
Fixed: https://github.com/cs96and/FoundryVTT-damage-log/pull/42/commits/3922d88427048bcf4e1794a1891c895a724f4820